core/evancore.vue.libs/src/components/permissions-editor/permissions-editor.ts

```diff
@@ -21,34 +21,40 @@ import Component, { mixins } from 'vue-class-component';
 
 // evan.network imports
 import { deepEqual, bccUtils } from '@evan.network/ui';
-import * as bcc from '@evan.network/api-blockchain-core';
-import { ContactInterface, ContainerPermissionsInterface } from '../../interfaces';
-import EvanComponent from '../../component';
-
+import { Runtime, Profile, ProfileOptions } from '@evan.network/api-blockchain-core';
 import { Prop, Watch } from 'vue-property-decorator';
+import {
+  ContactInterface,
+  ContainerPermissionsInterface,
+} from '../../interfaces';
+import EvanComponent from '../../component';
 
 const clone = (obj: any) => JSON.parse(JSON.stringify(obj)); // TODO: import from common utils
```

Comment:
Please use lodash deepClone.

```suggestion
```

Reply:
I didn't want to touch the logic in the component for now.